launcher_scripts/conf/rlhf_ppo/gpt3/2b_ppo.yaml

```diff
@@ -23,7 +23,7 @@ reward_model_server:
   pipeline_model_parallel_size: 1
   pipeline_model_parallel_split_rank: 0 # used for encoder and decoder model
   inference_micro_batch_size: 4
-  gpt_rm_model_file: "/lustre/fsw/joc/big_nlp/nemo_gpt3/rlhf/checkpoints/reward_models/from_sft/anthropic_helpful_all/2b_marigold_badger/megatron_gpt.nemo"
+  gpt_rm_model_file: ???  # path to trained reward model
```

Comment:
I'm not sure what happens if you set it to `null`. Since it won't work without a reward model I think it's better to use `???` to make it clear it is a required option.

Reply:
The CI will go and overwrite these attributes, I'm worried about these `???` causing issues there

Reply:
and the launcher actually loads these configs and writes out a new one. If `???` loads properly then I guess it's ok

Reply:
It should load but there are a few functions in OmegaConf that may not work until these values are set. If you can test it then we'll see whether or not this is an issue :)